packages/camera/camera_android/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Android implementation of the camera plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
 
-version: 0.10.8+13
+version: 0.10.8+14
```

Comment:
New functionality should increment the second version component, not the third.

Reply:
incremented the second version component (also for all platforms)